### DISCUSSION TITLE
Need a portable computer download version .

### DISCUSSION THREAD

**Discussion by Halchimer** [bridge-core/bridge.#281] [Q&A] [answered]
Hello, i only have a portable computer and i can't download bridge . I'm on Windows so can you add a windows portable computer download ( .zip file, but not the source code ) ? It must be cool !
P.S. : I'm french so sorry if my english is bad .  : )

**Comment by ENDERZOMBI102** [comment]
“portable” windows is normal windows, you only need the installer (.exe file) for your processor (x86, x64)

**Comment by Joelant05** [comment, marked as answer]
Here's a link to the exe download: https://github.com/bridge-core/bridge./releases/download/v1.7.9/bridge-web-setup-1.7.9.exe
